profile/README.md

```diff
@@ -1,7 +1,109 @@
 <pre>
       ____ _____  ____      Discover:  <a href="https://www.gno.land/">Homepage</a> - <a href="https://github.com/gnolang/gno">Main repo</a>
      / __ `/ __ \/ __ \     Networks:  <a href="https://test3.gno.land">Test3 (latest)</a> - <a href="https://staging.gno.land">Staging</a>
-    / /_/ / / / / /_/ /     Learn:     <a href="https://gno-by-example.com">Gno by Example</a> - <a href="https://github.com/gnolang/awesome-gno">Awesome Gno</a>
+    / /_/ / / / / /_/ /     Learn:     <a href="https://docs.gno.land">Official Documentation</a> - <a href="https://gno-by-example.com">Gno by Example</a> - <a href="https://github.com/gnolang/awesome-gno">Awesome Gno</a>
     \__, /_/ /_/\____/      Social:    <a href="https://twitter.com/_gnoland">Twitter</a> - <a href="https://discord.gg/tF2X8M6cVj">Discord</a> - <a href="https://reddit.com/r/gnoland">Reddit</a> - <a href="https://youtube.com/@_gnoland">YouTube</a>
    /____/                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 </pre>
+
+***
+<details><summary>Start</summary>
```

Comment:
Not sure it should be collapsed

Reply:
What do you mean, collapsed by default?

I'm not sure how else this would be done but we need the summary tags to encapsulate "Start" in order to prevent the default value "Details" being shown. Perhaps I am misunderstanding.

Reply:
I suggest this:

```
## Getting Started

<details><summary>Why Gno?</summary>

...

</details>

<details>...
```

Reply:
[7e63e94](https://github.com/gnolang/.github/pull/3/commits/7e63e94d78bfc4ad8d8f79b8be4f53c786c7e7b6)